src/test/apps.test.js

```diff
@@ -1090,3 +1091,69 @@ describe("App Status component", function () {
   });
 
 });
+
+describe("Breadcrumb Component", function () {
+  before(function () {
+    this.renderComponent = (group, app, task) => {
+      var renderer = TestUtils.createRenderer();
+      renderer.render(
+        <BreadcrumbComponent group={group} app={app} task={task} />
+      );
+      return renderer.getRenderOutput();
+    };
+  });
```

Comment:
Hmmm. The renderer is not re-used and no references to any of this persist outside the `describe` block - it doesn't feel like solid reasoning to me I'm afraid. 

Doesn't cost anything though so I'll make the change.